datasets/case01/queries/1104_column_disambiguation.sql

```diff
@@ -0,0 +1,9 @@
+-- Select the same column name from different tables
+-- Verifies that the columns are disambiguated to coexist in the
+-- results table, and re-ambiguated so that only the column name
+-- appears in the results returned to the user (as is normal
+-- mysql behavior)
+
+-- pragma sortresult
+
+SELECT Object.htmId20, Source.htmId20 FROM Object JOIN Source ON Object.objectId = Source.objectId;
```

Comment:
I don't think this unit test addresses the problem reported in the ticket:
```
SELECT *
FROM dp01_dc2_catalogs.object AS obj JOIN dp01_dc2_catalogs.truth_match AS truth
ON truth.match_objectId = obj.objectId
WHERE CONTAINS(POINT('ICRS', obj.ra, obj.dec), CIRCLE('ICRS', 61.863, -35.79, 0.05555555555555555))=1
AND (obj.mag_g <25 AND obj.mag_i <24)
```
Note the `SELECT *`.

Reply:
Actually, I was wrong - the query I posted above comes from the TAP service. So, please, ignore my question.